compose-builder/add-security.yml

```diff
@@ -225,94 +201,106 @@ services:
     image: ${CORE_EDGEX_REPOSITORY}/docker-security-proxy-setup-go${ARCH}:${CORE_EDGEX_VERSION}${DEV}
     container_name: edgex-proxy
     hostname: edgex-proxy
```

Comment:
Another place we need to fix naming for consistency. These should be `edgex-proxy-setup` and service should be `proxy-setup`

Reply:
fixed